static/ecosystem/consumers.tsx

```diff
@@ -87,12 +87,5 @@ export const Consumers: Array<Partner> = [
     description: "An OpenLineage connector collects OpenLineage events via a KafkaConsumer and transforms them into OpenMetadata Lineage edges.",
     docs_url: "https://github.com/open-metadata/OpenMetadata/pull/15317",
     org_url: "https://open-metadata.org/"
-  },
-  {
-    image: "snowflake_logo.svg",
-    org: "Snowflake",
-    description: "Snowflake's OpenLineage Adapter creates an account-scoped view from ACCESS_HISTORY and QUERY_HISTORY to output each query that accesses tables in OpenLineage JsonSchema specification.",
-    docs_url: "https://github.com/Snowflake-Labs/OpenLineage-AccessHistory-Setup",
-    org_url: "https://developers.snowflake.com/"
-  }
+  }, 
```

Comment:
Not sure about the comma here. But if it works on your local build then it's fine.

Reply:
the deploy-preview confirms that this works. Resolving